inc/html.class.php

```diff
@@ -6676,7 +6676,7 @@ public static function compileScss($args) {
                }
             } else {
                Toolbox::logDebug("$file is new, loading");
-               $GLPI_CACHE->set(md5($file), md5($path));
+               $GLPI_CACHE->set($fckey, md5(file_get_contents($path)));
```

Comment:
I think the best solution ot prevent difference on md5 is to move `$md5file = md5(file_get_contents($path));` from line 6669 to line 6666 and use `$GLPI_CACHE->set($fckey, $md5file);` here. By doing this, we would prevent computing the `$md5file` var are 2 different places.